docs/modules/ROOT/pages/contributor-guide/release-guide.adoc

```diff
@@ -233,13 +233,61 @@ Check the full [release announcement](https://camel.apache.org/blog/2021/06/came
 +
   * Click the green "Publish release" button at the bottom
 
+== Upgrade and tag Examples
+
+* Make sure all https://github.com/apache/camel-quarkus-examples/pulls[PRs] against `camel-quarkus-main` branch are merged.
+* Upgrade to the Camel Quarkus version you just released
++
+[source,shell]
+----
+$ NEW_CQ_VERSION=... # E.g. 2.2.0
+$ git fetch upstream
+$ git checkout camel-quarkus-main
+$ git reset --hard upstream/camel-quarkus-main
+$ find . -type f -name pom.xml -exec sed -i "s|<camel-quarkus.version>[^<]*</camel-quarkus.version>|<camel-quarkus.version>$NEW_CQ_VERSION</camel-quarkus.version>|g" {} \;
```

Comment:
Can't we do it with `./mvnw-for-each.sh versions:set` ?

Reply:
AFAIK `versions:set` can only set project version and cannot set a property version. But maybe there is another mojo able to that.

Reply:
ok, this was part of the misunderstanding. so, that's fine.